unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarsLODController.cs

```diff
@@ -0,0 +1,147 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL
+{
+    public class AvatarsLODController : MonoBehaviour
+    {
+        public static AvatarsLODController i { get; private set; }
```

Comment:
Just out of curiosity: why are we using a singleton for this? Shouldn't we add this `AvatarsLODController` into our `Environment`?

Reply:
I can try making that change

Reply:
+1. As the `CullingController` is already implemented using this approach would make total sense to have this there. We should avoid the `MonoBehaviour` as well.

Reply:
Agree!

Reply:
I went to the Assemblies Nightmare Forest and finally got out after 2 days, it's done!